### PR TITLE
Add kwargs to Classifier constructor

### PR DESCRIPTION
**Pull request by EPronovost**
## Title

fix(classifier): add kwargs to `Classifier.__init__`

## Type / Scope

- **Type**: Bug
- **Scope**: reward classifier

## Summary / Motivation

With the code currently in `main`, trying to train the reward classifier fails with 
```
TypeError: Classifier.__init__() got an unexpected keyword argument 'dataset_stats'
```

This PR adds `**kwargs` to `Classifier.__init__()` which are passed to the parent class. This is consistent with what other subclasses of `PreTrainedPolicy` do.

## What changed

- Add `**kwargs` to `Classifier.__init__()`

## How was this tested (or how to run locally)

Ran classifier training

## Checklist (required before merge)

- [ ] Linting/formatting run (`pre-commit run -a`)
- [ ] All tests pass locally (`pytest`)
- [ ] Documentation updated
- [ ] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

File: src/lerobot/policies/sac/reward_model/modeling_classifier.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 
 import logging
+from typing import Any
```
Comment (s1lent4gnt) [inline src/lerobot/policies/sac/reward_model/modeling_classifier.py:18]: Could you remove type annotation.

**Comment by s1lent4gnt** [comment]
@EPronovost Thank you for contribution!
Could you check my comment and update your branch?

**Comment by s1lent4gnt** [comment]
Hey @EPronovost, thank you for you contribution!
Superseded by #3075
